From dee8e9007cfcc4853bf6b0ead464c551ac857087 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Fri, 10 Aug 2012 15:50:24 +0800
Subject: ENGR00217918 - mx6 cpufreq : Add on-demand governor's threshold for
 FEC

Add on-demand governor's threshold for FEC to improves performance.
i.mx6q TO1.1 tx throughput only is 64Mbps in 100Mbps mode on sabresd
platform, after the change, the throughput can reach to 95Mbps for tx.

Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 arch/arm/mach-mx6/irq.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-mx6/irq.c b/arch/arm/mach-mx6/irq.c
index 2434c1b..22ca75c 100644
--- a/arch/arm/mach-mx6/irq.c
+++ b/arch/arm/mach-mx6/irq.c
@@ -81,6 +81,10 @@ static struct irq_tuner mxc_irq_tuner[] = {
 	 .up_threshold = 10,
 	 .enable = 1,},
 	{
+	 .irq_number = 150, /* ENET */
+	 .up_threshold = 4,
+	 .enable = 1,},
+	{
 	 .irq_number = 0, /* END */
 	 .up_threshold = 0,
 	 .enable = 0,},
-- 
1.7.9.5

